src/components/dropdown-menu/DropdownMenu.tsx

```diff
@@ -32,20 +32,20 @@ interface DropdownMenuClickProps<T> extends DropdownMenuBaseProps<T> {
   trigger: 'click';
 }
 
-interface DropdownMenuContextProps<T> extends DropdownMenuBaseProps<T> {
+interface DropdownMenuContextProps<T, E> extends DropdownMenuBaseProps<T> {
   trigger: 'contextMenu';
   children: ReactNode;
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  as?: ElementType | React.ComponentType<any>;
+  as?: E | React.ComponentType<any>;
```

Comment:
Why keep `React.ComponentType`?

Reply:
it is a mistake, I Just replace the name.